### ISSUE TITLE
Getting $collStat for views causes errors

### ISSUE THREAD

**Issue by nayavu** [percona/mongodb_exporter#502]
**Describe the bug**
With discovering mode enabled, while getting the list of collections, the exporter does not distinguish collections from views and then tries to get `$collStat` for views. It is an invalid operation on a view and as a result both Exporter logs and MongoDB logs are flooded with errors (`CommandNotSupportedOnView`)

**Logs**
An example of MongoDB log:
```
{"t":{"$date":"2022-06-13T23:47:37.171+00:00"},"s":"W",  "c":"COMMAND",  "id":23799,   "ctx":"conn4","msg":"Aggregate command executor error","attr":{"error":{"code":166,"codeName":"CommandNotSupportedOnView","errmsg":"Namespace testdatabase.test-view is a view, not a collection"},"stats":{},"cmd":{"aggregate":"ncbi-urls","pipeline":[{"$collStats":{"latencyStats":{"histograms":false},"storageStats":{"scale":1}}},{"$project":{"storageStats.wiredTiger":0,"storageStats.indexDetails":0}}],"cursor":{},"lsid":{"id":{"$uuid":"a76402dd-473a-4ad9-acea-b7b8c042c03b"}},"$db":"testdatabase"}}}
```

**Environment**
- MongoDB Exporter v 0.32.0
- MongoDB 5.0.9, 
- Amazon Linux 2 (EC2)

**Comment by ShashankSinha252** [comment]
@nayavu , thanks for reporting the issue. This looks like a valid bug. I'll take a look at it.

**Comment by vCra** [comment]
Hey @ShashankSinha252 @denisok I think I have fixed this issue in #626. Let me know what the next steps for a review is

**Comment by vCra** [comment]
Looks like this has been fixed by https://github.com/percona/mongodb_exporter/pull/790
